sdk/test-utils/test-credential/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Release History
 
+## 2.1.0 (Unreleased)
+
+### Breaking Changes
+
+Updates the `createTestCredential` method to consume `DefaultAzureCredential` instead of `ClientSecretCredential` in order to offer autonomy to the devs and to move away from clietn secrets in environment varaibles.
```

Comment:
nit
```suggestion
Updates the `createTestCredential` method to consume `DefaultAzureCredential` instead of `ClientSecretCredential` in order to offer autonomy to the devs and to move away from client secrets in environment varaibles.
```

Reply:
oops